src/Viewer/components/Monaco/MonacoInstance.js

```diff
@@ -100,21 +98,17 @@ function MonacoInstance ({
 }) {
     const {theme} = useContext(ThemeContext);
     const editorRef = useRef(null);
-    const monacoRef = useRef(null);
     const timeoutRef = useRef(null);
-    const [monacoTheme, setMonacoTheme] = useState("customLogLanguageLight");
-
-    const [language, setLanguage] = useState("");
-
-    loader.config({monaco});
+    const editorContainerRef = useRef(null);
+    const getMonacoThemeName = (theme) => (
+        (theme === THEME_STATES.LIGHT) ? "customLogLanguageLight" : "customLogLanguageDark"
+    );
 
     /**
      * Called before the monaco editor is mounted.
-     *
-     * @param {object} monaco
      */
-    function handleEditorWillMount (monaco) {
-        beforeMount(monaco);
+    const handleEditorWillMount = () => {
```

Comment:
Consider renaming this to `_beforeMount` so we are consistent with the name in the component props.